src/control/Control.ts

```diff
@@ -16,6 +18,12 @@ abstract class Control {
   protected _controller: AxesController;
   protected _activePanel: Panel | null;
 
+  public get controller() { return this._controller; }
+  public get activeIndex() { return this._activePanel?.index ?? -1; }
```

Comment:
Does the meaning of `-1` mean there is no active index? If so, I think you have to some add description.

Reply:
As a flicking returns `-1` from `getIndex` when there are no panels that exist, that also returns `-1` when there is no active panel.
This is documented on the Flicking's `getIndex`, but not on the Control as there's no description on the Control yet.
I'll add the descriptions while adding the whole documentation of it :)